aries-site/src/pages/index.js

```diff
@@ -1,69 +1,118 @@
-import { Layout, PageIntro } from '../layouts';
-import { getCards, getPageDetails } from '../utils';
+import PropTypes from 'prop-types';
+
+import { Meta } from '../components';
+import {
+  Community,
+  Featured,
+  Highlights,
+  Quote,
+  WhatIs,
+} from '../components/home';
+import { Layout /* , PageIntro */ } from '../layouts';
+import { getPageDetails } from '../utils';
 
 const title = 'Home';
 const pageDetails = getPageDetails(title);
-const cards = getCards();
-const featured = [
-  'Designer Guidance',
-  'Forms',
-  'Header',
-  'Human Centered',
-  'Developer Guidance',
-  'Button',
-  'Navigation',
-  'Icons',
-  'Search',
-  'Typography',
-];
 
-const featuredCards = [];
-const cardOrder = [];
+const calcPad = size => {
+  const val = size !== 'small' ? 'xlarge' : 'large';
+  return val;
+};
+
+const Intro = ({ children }) => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Box height={{ min: 'medium' }} justify="center">
+      <Grid gap="large" columns={size === 'small' ? ['auto'] : ['3/4', 'auto']}>
+        <Card background="none" elevation="none">
+          {children}
+        </Card>
+        {size !== 'small' && (
+          <Card background="none" elevation="none" height="small" />
+        )}
+      </Grid>
+    </Box>
+  );
+};
 
-cards.forEach(card => {
-  if (featured.includes(card.name)) featuredCards.push(card);
-  else cardOrder.push(card);
-});
+Intro.propTypes = {
+  children: PropTypes.node,
+};
 
-featuredCards.sort(
-  (a, b) => featured.indexOf(a.name) - featured.indexOf(b.name),
-);
+const Index = () => {
+  const size = useContext(ResponsiveContext);
+  const videoRef = useRef(null);
 
-cardOrder.splice(0, 0, ...featuredCards);
+  const [videoEnabled, setVideoEnabled] = useState(false);
+  const backgroundImage = {
+    src: { dark: '/home.svg', light: '/home.svg' },
+    alt: 'HPE Design System',
+    margin: { top: '75px', left: '0px' },
+    style: {},
+    position: size !== 'small' ? 'right' : 'left',
+  };
 
-const Index = () => (
-  <Layout
-    backgroundImage={{
-      src: { dark: '/cards-dark.svg', light: '/cards.svg' },
-      alt: 'HPE Design System',
-      margin: { top: '50px', left: '-150px' },
-      style: { transform: 'scale(1.4)', transformOrigin: 'top left' },
-      small: {
-        margin: { left: '-75px', top: '-75px' },
-      },
-      useGrid: true,
-    }}
-    title={title}
-    isLanding
-  >
-    <Meta title={title} description={pageDetails.seoDescription} />
-    <Box gap="large">
-      <PageIntro>
-        <Box justify="center" fill>
-          <Heading margin="none">HPE Design System</Heading>
-          <Paragraph size="large" fill>
-            The Design System was created to empower designers, developers, and
-            others to contribute to making great experiences for the customer.
-          </Paragraph>
+  return (
+    <Layout backgroundImage={backgroundImage} title={title} isLanding pad={{}}>
```

Comment:
Thinking about how this image scales and wondering if we should try approaching from a different angle?

Wondering about moving the image away from the Layout's backgroundImage and instead presenting the image in the Intro section. The current homepage design calls for the card images to break the page gutters/design grid which makes sense to have the image tied to Layout. However, the new image and animation doesn't seem to have that same need. So, it might be easier to get the intro heading and image to move together more harmoniously by moving the image into the "ghost/spacing Card". 

Thoughts?

Reply:
I think that would be easier to make it behave well with the text. What I'm not sure about is how much should the image overflow into the header, margin and next section like the other pages do. I assume that's basically why it's done in the layout as a stack layer.

Reply:
I implemented something like this in the most recent version. Let me know what you think.